### DISCUSSION TITLE
登录页报500

### DISCUSSION THREAD

**Discussion by chenguod** [lsky-org/lsky-pro#785] [问答]
大佬好，图床内存储的照片可以正常访问，但是登录页显示500，使用docker部署，报错时docker日志内容如下：
192.168.68.69 - - [21/Mar/2024:15:09:32 +0000] "GET /login HTTP/1.1" 500 211 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"

日志内容如图：https://images.chenmx.net/blog/20240321215538.png

**Comment by WuMe-sicx** [comment]
正常情况下看这个地方的日志/storage/logs
